index.html

```diff
@@ -0,0 +1,198 @@
+        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAAXNSR0IArs4c6QAAAK1JREFUaEPtlkEKwCAQxPT/j64/kIEwLNL07Ky7iWL3evzbj/e/HGDaoAY0AAl4hCBAHNcARggL/MLAByHR+BVyYsABoAINQIDdeHIHuh3A6g4AAeK4BjBCWCAx4EPWhKwBSDeJ41+JZJOxNckRGmsu2dgBEkrNNRpo0k1qJwZ8iROSlzX4HdDAtAG4fzeeXOJuB7C6A0CAOK4BjBAW0AAEiOMawAhhAQ1AgDh+AJfZDDGRtEaJAAAAAElFTkSuQmCC" alt="logo" class="logo-translate"/>
+      </div>
+      <div class="nav-link-container">
+        <a>Live</a>
+        <a>Push</a>
+        <a>Note</a>
+        <a>Link</a>
+        <a>Shop</a>
+        <a>Packs</a>
+        <a>Help</a>
+        <a class="nav-link-more">More +</a>
+      </div>
+    </div>
+    <div class="nav-right">
+      <a class="try-free">Try Live for free</a>
+      <a class="auth">Log in or register</a>
+    </div>
+  </nav>
+  <nav class="secondary-navbar">
+    <a class="about">About</a>
+    <a>Jobs</a>
+    <a>Apprenticeships</a>
+  </nav>
+<!-- =========================First section=========================== -->
+  <section class="first-section">
+    <div class="image-container">
+      <img src="./img/ableton.jpg" alt="ableton image" />
+      <h1>Ableton</h1>
+    </div>
+    <div class="text-container">
+      <h3>We make <a style="color:#0000ff;">Live</a>, <a style="color:#0000ff;">Push</a> and <a style="color:#0000ff;">Link</a> — unique software and hardware for music creation and performance. With these products, our community of users creates amazing things.</h3>
+      <p>Ableton was founded in 1999 and released the first version of Live in 2001. Our products are used by a community of dedicated musicians, sound designers, and artists from across the world.</p>
+    </div>
+  </section>
+<!-- =========================Second section=========================== -->
+  <section class="second-section">
+    <div class="image-container">
+      <div class="secondary-image-container">
+        <img src="./img/building.jpg" alt="building image" class="building">
+        <img src="./img/chair.jpg" alt="chair image" class="chair">
+      </div>
+    </div>
+    <div class="text-container">
+      <h3>Making music isn’t easy. It takes time, effort, and learning. But when you’re in the flow, it’s incredibly rewarding.</h3>
+      <p>We feel the same way about making Ableton products. The driving force behind Ableton is our passion for what we make, and the people we make it for.</p>
+    </div>
+  </section>
+  <!-- =========================Third section=========================== -->
+  <section class="third-section">
+    <div class="image-container">
+      <iframe  src="https://www.youtube.com/embed/DNJ5JqLw9YM?si=0mS_DCE_RIMDU-_c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
pls check this link 
<img width="547" alt="Screen Shot 2566-12-08 at 08 49 29" src="https://github.com/siripoom-datawow/Ableton-HTML-CSS/assets/106588699/fb450785-b1e8-4e7c-b232-ef2417917266">